pkg/migration/controllers/migration.go

```diff
@@ -402,6 +402,10 @@ func (m *MigrationController) handle(ctx context.Context, migration *stork_api.M
 					}
 					return nil
 				}
+
+				// to run transformation validation on remote cluster
+				resp.Spec.ClusterPairName = migration.Spec.ClusterPair
```

Comment:
I think we should probably avoid doing a dry-run during Migration. This adds to more "GetResourceForType" k8s API calls. If the initial validation was successful, then lets just try the transformation and if it fails then fail the Migration.  Any way if the Transform CR dry-run validation fails here we fail the migration here.